src/components/details-view/details-view.js

```diff
@@ -174,6 +184,7 @@ class DetailsView extends Component {
       // if bottoming out, enable isSticky
       if (bottomedOut && !isSticky && isListScrollbarNeeded) {
         this.setState({ isSticky: true });
+        this.$sticky.scrollIntoView();
```

Comment:
I added this to scroll up to titles list accordion so it always stays fully in view when a user has scrolled down to it. Without this there are cases when a user scrolls down very fast and a part of accordion is not visible